### DISCUSSION TITLE
Lighthouse throws warning for the duplicate javascript

### DISCUSSION THREAD

**Discussion by shivgarg5676** [vercel/next.js#21192] [Help]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

12.18.2

**What browser are you using?**

chrome

**What operating system are you using?**

macos

**How are you deploying your application?**

NODE_ENV='production' next start

**Describe the Bug**

This is a link to my website https://staging-website.azent.com/. When I test it on lighthouse I get the warning that page contains duplicate javascript. 
I think this is due to rel=preload on the script in the head section and same link in the end of the body without rel=preload.

**Expected Behavior**

The page should not contain any link for the same javascript module.

**To Reproduce**

https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fstaging-website.azent.com%2F
Visit this link  to check page performance

https://staging-website.azent.com/ for the actual website.

Any suggestions or workarounds will also be fine.

![image](https://user-images.githubusercontent.com/6873680/104115755-a8c9bf00-5338-11eb-8fba-08db039c0d2d.png)
![image](https://user-images.githubusercontent.com/6873680/104115761-b717db00-5338-11eb-9cf4-3b7fe1e316eb.png)
![image](https://user-images.githubusercontent.com/6873680/104115768-c139d980-5338-11eb-9fa7-eda9fa822b48.png)

**Comment by macstr1k3r** [comment]
the issue is also present on the Next.js landing page
![image](https://user-images.githubusercontent.com/354308/107148763-41ae2300-6955-11eb-8bf2-0bc9d103f910.png)

**Reply by shivgarg5676** [reply]
yes, it is definitely there. We tried asking for a fix here 
https://github.com/vercel/next.js/issues/7392
https://github.com/vercel/next.js/issues/20932
But nobody gave a damn to it :(

**Reply by xboy2012** [reply]
I opened [#7392](https://github.com/vercel/next.js/issues/7392), and can verify that **It's fixed in nextjs 8.x**

How ever it might be a version management problem.

My PR is merged into the 8.x branch.
However the 9.x 10.x and so on does not get fixed.

My suggestion is to create a PR for every major version branch to fix that, util the latest version is fixed.

Anyone helps?
